src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "./utils/function";
+@import "./blocks/reset.scss";
```

Comment:
It's recommended to use underscore prefix for SCSS partial files like '_reset.scss' and '_calendar.scss'.